admin/services/data-access/check-form.data.service.js

```diff
@@ -0,0 +1,26 @@
+'use strict'
+
+const CheckForm = require('../../models/check-form')
+
+const checkFormDataService = {
+  /**
+   * Get form by id, when isDeleted is false.
+   * @param id
+   */
+  getActiveForm: (id) => {
+    return CheckForm.findOne({_id: id, isDeleted: false})
```

Comment:
This really really needs to return the JSON object.  So `.lean().exec()` is required here.